app/views/boards/_board.html.erb

```diff
@@ -11,85 +11,94 @@
+                  <%= link_to project_board_list_card_path(current_project, board, list, card) do %>
+                    <div class="card-details">
+                      <div class="card-title"><%= card.name %></div>
+                      <div class="card-badges">
+                        <% if card.due_date %>
+                          <% case (card.due_date - Date.today).to_i %>
+                          <% when -Float::INFINITY..-1 %>
+                          <div class="badge-due-date badge badge-danger">
+                          <% when 0 %>
+                          <div class="badge-due-date badge badge-warning">
+                          <% else %>
+                          <div class="badge-due-date badge badge-success">
+                          <% end %>
+                            <span><i class="fa fa-clock-o"></i></span>
+                            <span><%= card.due_date.strftime('%b %-d') %></span>
+                          </div>
                         <% end %>
-                          <span><i class="fa fa-clock-o"></i></span>
-                          <span><%= card.due_date.strftime('%b %-d') %></span>
-                        </div>
-                      <% end %>
-                    </div>
-                    <div class="card-members">
-                      <% card.assignees.each do |user| %>
-                        <%= avatar_image(user, size: 30) %>
-                      <% end %>
+                      </div>
+                      <div class="card-members">
+                        <% card.assignees.each do |user| %>
+                          <%= avatar_image(user, size: 30) %>
+                        <% end %>
+                      </div>
                     </div>
-                  </div>
-                <% end %>
-              </li>
-              <% end %> <%# board-list-card cache %>
-            <% end %>
-          </ul>
-          <%= link_to 'Add a task...', new_project_board_list_card_path(current_project, board, list), class: 'card-new' %>
-        </li>
-        <% end %> <%# board-list cache %>
-      <% end %>
+                  <% end %>
+                </li>
+                <% end %> <%# board-list-card cache %>
+              <% end %>
+            </ul>
+            <%= link_to 'Add a task...', new_project_board_list_card_path(current_project, board, list), class: 'card-new' %>
+          </li>
+          <% end %> <%# board-list cache %>
+        <% end %>
 
-      <li class="list list-new">
-        <a
-          class="js-list-modal"
-          href="#modal-list-new"
-        >
-          Add a list...
-        </a>
-      </li>
-    </ul>
+        <li class="list list-new">
+          <a
+            class="js-list-modal"
+            href="#modal-list-new"
+          >
+            Add a list...
+          </a>
```

Comment:
I realize this isn't a change in the PR but I just have to point at this and ask the world why 😄